Device.md

```diff
@@ -434,7 +426,7 @@ gantt
     0      :d1, 0, 1
 ```
 
-An array of 8 bytes that can be used to store a tag for a specific firmware version. For instance, it could be used to store the Git hash of a specific release/commit. The byte-order is little-endian. This register is read-only.
+An array of 8 bytes that can be used to store a tag for a specific firmware build. For instance, it could be used to store the Git hash of a specific release/commit. The byte-order is little-endian. This register is read-only.
```

Comment:
```suggestion
An array of 8 bytes that can be used to store firmware-specific metadata. For instance, it could be used to store the Git hash of a specific release/commit. The byte-order is little-endian. This register is read-only.
```

Is this a "build" id, or a Git commit hash? The build ID is usually associated with knowing exactly when and where the build was generated, regardless of the source-code version but I don't think this is what we mean here.

"Version" would work technically for a commit hash in a version-control system, but I understand the confusion with the notion of official "release", so I tried to reword it to avoid clashing.